tests/model_registry/model_catalog/test_custom_model_catalog.py

```
@@ -55,7 +55,7 @@
 class TestModelCatalogCustom:
     def test_model_custom_catalog_list_sources(
         self: Self,
-        updated_catalog_config_map: tuple[ConfigMap, str, str],
+        enabled_model_catalog_config_map: ConfigMap,
```

Ruff (0.14.7)

Unused method argument: `enabled_model_catalog_config_map`

(ARG002)